swampmachine-multimedia/src/main/java/net/kiberion/audio/AudioService.java

```diff
@@ -23,20 +20,27 @@
     @Autowired
     public void setMixer(Mixer mixer) {
         this.mixer = mixer;
-        musicChannel = mixer.newChannel(MUSIC_CHANNEL_ID);
-        soundChannel = mixer.newChannel(SOUND_CHANNEL_ID);
     }
     
     @EventListener
     public void playSound (PlaySoundEvent event) {
         log.info("Play sound");
-        soundChannel.playSound(soundRegistry.getSoundEntries().get(event.getSoundId()));
+        mixer.getChannel("sound").playSound(soundRegistry.getSoundEntries().get(event.getSoundId()));
```

Comment:
It is better. It says directly what it needs to without CHANNEL_ID clutter and a needless constant. Using constants for this kind of things makes sense if you need them to be integers or if you're using protocol which might change - but neither is the case here.

That's not to mention that it is going to be removed out of there anyway cause users should be able to choose channel for themselves.